ci/ray_ci/docker_container.py

```diff
@@ -83,8 +40,11 @@ def _get_image_version_tags(self) -> List[str]:
 
         return [sha_tag]
 
-    def _get_image_names(self) -> List[str]:
-        # Image name is composed by ray version tag, python version and platform.
+    def _get_canonical_tag(self) -> str:
+        return self._get_image_tags()[0]
```

Comment:
could you add a comment on why this is always not empty and what is special with the first tag?

and add a unit test for this function?